### PR TITLE
release 1.2.1

### PR DESCRIPTION
**Pull request by marsishandsome**
close https://github.com/pingcap/tispark/issues/900

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 setup(
     name = 'pytispark',
     packages = ['pytispark'],
-    version = '1.1',
+    version = '1.2.1',
```
Comment (birdstorm) [inline python/setup.py]: if you update pytispark version, it should be released on pyti

Comment (marsishandsome) [reply python/setup.py]: shall we release pytispark?

Comment (birdstorm) [reply python/setup.py]: we may need to release pytispark when changes involve in logic or spark version.

Comment (marsishandsome) [reply python/setup.py]: reverted

**Comment by marsishandsome** [comment]
@birdstorm PTAL

**Pull request merged**